client/js/libs/handlebars/parse.js

```diff
@@ -83,6 +91,9 @@ module.exports = function parse(text) {
 			return `<span class="inline-channel" role="button" tabindex="0" data-chan="${escapedChannel}">${fragments}</span>`;
 		} else if (textPart.emoji) {
 			return `<span class="emoji">${fragments}</span>`;
+		} else if (textPart.nick) {
+			const nick = fragments.trim();
```

Comment:
Why is it trimmed?

Reply:
Was an issue from before with handlebars, need to test if necessary now